build-packages.cmd

```diff
@@ -1,18 +1,47 @@
 @if "%_echo%" neq "on" echo off
-setlocal
+setlocal EnableDelayedExpansion
 
 set packagesLog=build-packages.log
 echo Running build-packages.cmd %* > %packagesLog%
 
-if /I [%1] == [/?] goto Usage
+set native=false
+set options=/nologo /v:minimal /flp:v=detailed;Append;LogFile=%packagesLog%
+
+:Loop
+if [%1]==[] goto Begin
+
+if /I [%1]==[/?] goto Usage
 if /I [%1] == [/help] goto Usage
 
+if /I [%1] == [/n] (
+  set native=true
+  goto Next
+)
+
+if /I [%1] == [/native] (
+  set native=true
+  goto Next
+)
+
+echo Unrecognized argument '%1'
```

Comment:
Agree. 
For this particular case, I'll delete the /native from the scripts and once we have the documentation of the dev workflow, I'll include this option there.